Contrib/MLpackages/GlobalAncestry.py

```diff
@@ -0,0 +1,147 @@
+#Global Ancestry with 1k genomes
+
+import pandas as pd
+import numpy as np
+import torch
+import sys
+
+from sklearn import datasets, linear_model
+from sklearn.metrics import mean_squared_error, r2_score
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.linear_model import LinearRegression
+
+#Load the vcf file and ancestry information file
+#genofile = '1kGPfiltered/pruned500k1kGP.vcf'
+#AncstFile = 'igsr_IDwAncst_subset-Copy1.txt'
+#AncstFile = 'igsr_IDwAncst.txt'
+genofile = open(sys.argv[1],'r')
+AncstFile = open(sys.argv[2],'r')
+log = open(sys.argv[3],'w')
+filename = sys.argv[4]
+
+
+
+#Read in both files
+headervcf=pd.read_csv(genofile,delimiter='\t',skiprows=25)
+IDswAncst = pd.read_csv(AncstFile,delimiter = ',')
+
+
+
+#Turn genotypes into dosage format for PCA
+dosagevcf = headervcf.replace('0/0', 0)
+dosagevcf = dosagevcf.replace('0/1', 1)
+dosagevcf = dosagevcf.replace('1/0', 1)
+dosagevcf = dosagevcf.replace('1/1', 2)
+
+
+#Separate the genotype dosages
+IDs = headervcf.iloc[1]
```

Comment:
slightly surprising that we're referring to both headervcf and dosagevcf here, since one was derived from the other and I wouldn't expect the transformation we applied to affect the IDs [?]